### PR TITLE
Faq

### PR DESCRIPTION
**Pull request by kongsally**
r @apiatski @sanjayss34

### CODE REVIEW

File: app/templates/main/faq.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        <p>No, in fact, if you idle for longer than 10 seconds you are wasting more gas. It used to be true, with older vehicles that had carburetors, that turning an engine on would consume substantial amounts of fuel. However, modern vehicles (anything past the mid-80s) use fuel-injection and this uses almost no fuel to start an engine.</p><br>
+
+        <h3>Is idling a vehicle the best way to get it ready to drive ("warmed-up")?</h3>
+        <p>No, idling causes unnecessary wear and tear on your vehicle. The best way, by far, to get the engine warmed up is to drive it. Modern engines respond best when driven, not sitting idling.</p><br>
+
+        <h3>What about getting the interior warm or cold, is idling ok for that?</h3>
+        <p>No, driving the vehicle is the quickest way to get warm or cold air into the interior of the vehicle.</p><br>
+
+        <h3>Does idling really waste that much gas?</h3>
+        <p>Yes, idling a vehicle can waste between 1/5 - 7/10 of a gallon of gas per hour.</p><br>
```
Comment (sanjayss34) [inline app/templates/main/faq.html:17]: Would percentages have better effect here, or do you think that fractions are the most communicative?

**Comment by sanjayss34** [comment]
The code looks good to me!

**Comment by sandlerben** [comment]
I could have sworn I already left this comment? There are duplicate commit names in the commit history. Please clean :bath:

**Comment by sandlerben** [comment]
Also, now realizing that most of this presumably doesn't have to directly do with the faq page. Can you better describe what this PR is about? 

I suspect that all of your changes from the base styles branch are also here. You only want your commits dealing with the faq page here.

This also explains my confusion above! I commented on #26!

**Pull request closed without merging**